### PR TITLE
bump django version in setup.py

### PR DESCRIPTION
**Pull request by fericsepi**
To do (delete all that do not apply):

 - [ ] Change has a jira ticket that has the correct status.
 - [ ] Changelog entry added.
 - [ ] (if there are vulnerable requirements) Upgraded any vulnerable dependencies.
 - [ ] (if updating requirements) Requirements have been compiled.
 - [ ] (if adding env vars) Added any new environment variable to vault.
 - [ ] (if adding feature flags) Cleaned up old flags

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
 setup(
     name='directory_client_core',
-    version='6.2.0',
+    version='6.2.1',
```
Comment (depsiatwal) [inline setup.py:9]: Can we bump minor version please as we normally use patch for small bug fixes

**Review by depsiatwal — COMMENTED** [review]
Change log entry please

File: setup.py (hunk 1 above)
Comment (fericsepi) [reply setup.py:9]: ok

**Comment by fericsepi** [comment]
tests are stuck, let's try to open another PR

**Pull request closed without merging**